content/docs/start/experiments.md

```diff
@@ -54,92 +128,84 @@ reflected in the experiment run.
 
 ```dvc
```

Comment:
If we end up introducing `dvc exp show` above, I think we could remove `dvc exp diff`. It's mostly a less powerful version of `dvc exp show`.